### PR TITLE
Update Workflows to latest version of Microsoft Teams

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This PR updates the following to the latest version of Microsoft Teams: 

Block_Host_with_Fortinet_Firewall_from_Microsoft_Teams
Block_Host_with_Palo_Alto_Firewall_from_Microsoft_Teams
Block_Host_with_Check_Point_Firewall_from_Microsoft_Teams
Block_Sender_in_Office_365_with_Teams
Check_Host_Block_Status_with_Check_Point_Firewall_from_Microsoft_Teams
Check_Host_Block_Status_with_Fortinet_Firewall_from_Microsoft_Teams
Check_Host_Block_Status_with_Palo_Alto_Firewall_from_Microsoft_Teams

## Testing / Validation: 
Each workflow was validated using ICON-Validate. Each workflow was imported into ICON.

### CODE REVIEW

File: workflows/Block_Host_with_Palo_Alto_Firewall_from_Microsoft_Teams/help.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
+* 1.1.1 - Update Microsoft Teams to latest version
```
Comment (mhofert-r7) [inline workflows/Block_Host_with_Palo_Alto_Firewall_from_Microsoft_Teams/help.md:71]: Lets consistently use `Update to make Microsoft Teams plugin the latest version`

File: workflows/Check_Host_Block_Status_with_Fortinet_Firewall_from_Microsoft_Teams/help.md [hunk 2]
```diff
@@ -36,14 +36,15 @@ Plugins utilized by workflow:
...
-|Microsoft Teams|2.0.3|3|
+|Microsoft Teams|2.0.4|3|
 
 ## Troubleshooting
 
 _There is no troubleshooting information at this time_
 
 # Version History
 
+* 1.1.1 - Update Microsoft Teams to the latest version
```
Comment (mhofert-r7) [inline workflows/Check_Host_Block_Status_with_Fortinet_Firewall_from_Microsoft_Teams/help.md:47]: see above

File: workflows/Check_Host_Block_Status_with_Check_Point_Firewall_from_Microsoft_Teams/help.md [hunk 3]
```diff
@@ -33,16 +33,17 @@ Plugins utilized by workflow:
 
 |Plugin|Version|Count|
 |----|----|--------|
-|Check Point NGFW|2.0.0|1|
+|Check Point NGFW|2.0.1|1|
```
Comment (mrinehart-r7) [inline workflows/Check_Host_Block_Status_with_Check_Point_Firewall_from_Microsoft_Teams/help.md:36]: Did you mean to bump this one?

Comment (jmcadams-r7) [reply workflows/Check_Host_Block_Status_with_Check_Point_Firewall_from_Microsoft_Teams/help.md:36]: Yes. It failed validation.

File: workflows/Block_Host_with_Palo_Alto_Firewall_from_Microsoft_Teams/help.md (hunk 1 above)
Comment (jmcadams-r7) [reply workflows/Block_Host_with_Palo_Alto_Firewall_from_Microsoft_Teams/help.md:71]: Fixed

File: workflows/Check_Host_Block_Status_with_Fortinet_Firewall_from_Microsoft_Teams/help.md (hunk 2 above)
Comment (jmcadams-r7) [reply workflows/Check_Host_Block_Status_with_Fortinet_Firewall_from_Microsoft_Teams/help.md:47]: Fixed

**Pull request merged**